hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -331,46 +327,28 @@ initialize ctx =
  where
   ChainContext{networkId} = ctx
 
--- | Construct a commit transaction based on known, spendable UTxO and some
--- arbitrary UTxOs to commit. This does look for "our initial output" to spend
--- and check the given 'UTxO' to be compatible. Hence, this function does fail
--- if already committed or if the head is not initializing.
---
--- NOTE: This version of 'commit' does only commit outputs which are held by
--- payment keys. For a variant which supports committing scripts, see `commit'`.
-commit ::
```

Comment:
I would suggest NOT to remove this as it is widely used and we can redefine this easily in terms of the more generic blueprint one. (Also likely want to keep the one taking `(TxOut CtxUTxO, Witness WitCtxTxIn)`)